README.md

```diff
@@ -2,15 +2,47 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Testing in each PR
+## Cointinuous Deployment and commit format
+
+We use [semantic release](https://github.com/semantic-release/semantic-release) to generate each tag for automatic versioning, that's why it's important to have each commit formatted correctly, this tool uses [Angular commit message](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format) format by default.
+The format is the following:
+
+`<type>(<scope>): <short summary>`
+
+The types we most use are:
+
+- docs: Documentation only changes
```

Comment:
I have never use this commit type, but maybe this should be appropriated for yours instead use `fix`

Reply:
But docs wouldn't ship right away right?

Reply:
I don't get you 🤔

Reply:
I've added all changes as a fix because I'm fixing some missing information in readme, it could be also a doc, but doc type will not trigger a version change.

Reply:
Do we need to trigger a version change for this?

Reply:
I would love to have this deployed as soon as possible to forget about this. 😄

Reply:
I'm used to have this deployed, you are right, the code is hosted and viewed here at github so it is the same. Perhaps I shoud try doc.